app/controllers/plans_controller.rb

```diff
@@ -1,17 +1,13 @@
 class PlansController < ApplicationController
   require 'pp'
   helper SettingsTemplateHelper
-
+  helper PaginableHelper
```

Comment:
Can you not just include the Concern directly?

Reply:
We need a helper since the concern method (e.g. paginable_renderise) is only available as an instance method for the controller paginable/plans_controller but it is used in the view (views/plans/index.html) so therefore a helper is needed.

Reply:
ah ok. didn't realize that